### ISSUE TITLE
More tutorials ?

### ISSUE THREAD

**Issue by iamshulz** [luvejo/vue-3-carousel-tutorial#1]
In case if you are interested in making more tutorial (I really hope so)

Can you do tutorial where we create a tag on an input and whenever we press enter a new tag will be added but if there are too many tags in the input the others will not be shown

**Comment by luvejo** [comment]
Hi, Oliver. Thanks for your kind words. Do you have an example of how that component would look like?

**Comment by jamols09** [comment]
Somewhat like this I can also set the maximum number of tags so the input field would not expand when there are many tags. https://github.com/vueform/multiselect

**Comment by jamols09** [comment]
You're right think that is the main structure (also from the video I've seen).
I think the more complicated part is the making of dropdown ? (probably) because it has to be done from scratch and also the dynamic width input width size.

**Comment by petrasvestartas** [comment]
Does it support videos?

**Comment by larxn** [comment]
Hi, @petrasvestartas. You mean like thumbnails or actual video players? I think it should work on both cases, but haven't tried.

I would recommend you to use a professional grade library though; something like [swiper.js](https://swiperjs.com/).

**Comment by petrasvestartas** [comment]
Yesterday night I tried swiper, it worked quite well with version of swiper 6. Thank you for reference.
